Jenkinsfile

```
@@ -12,12 +12,13 @@ properties([
 
 // Define the sequential stages and the parallel steps inside each stage
 def sequentialStages = [:]
-sequentialStages['Tool'] = [ 'java', 'maven', 'maven-windows', 'maven-11', 'maven-11-windows']
-sequentialStages['OS & Java'] = [ 'linux', 'windows', 'jdk8', 'jdk11']
+sequentialStages['Tool'] = [ 'java', 'maven', 'maven-11', 'maven-windows', 'maven-11-windows']
+sequentialStages['OS & Java'] = [ 'linux', 'jdk8', 'jdk11', 'windows']
 sequentialStages['Processor'] = [ 'arm64', 'amd64' ] // Remove ppc64le and s390x until virtual machine available 'ppc64le', 's390x'
-sequentialStages['Docker'] = [ 'arm64docker', 'docker', 'docker-windows' ] // Remove ppc64le and s390x until available again 'ppc64ledocker', 's390xdocker'
+sequentialStages['Docker'] = [ 'arm64docker', 'docker', 'windows'] // Remove ppc64le and s390x until available again 'ppc64ledocker', 's390xdocker'
```

As far as I understand it, we need a label that means "docker for Windows containers".  I think that we should be checking 

* "docker for aarch64 linux containers" with the "arm64docker" label
* "docker for amd64 linux containers" with the "docker" label
* "docker for amd64 windows containers"  with the "docker-windows" label
```suggestion
sequentialStages['Docker'] = [ 'arm64docker', 'docker', 'docker-windows'] // Remove ppc64le and s390x until available again 'ppc64ledocker', 's390xdocker'
```

I think this change is required, otherwise we will stop checking that we can allocate containers for Windows.

---

this is clearly a mistake, I didn't plan to change the agent list ...